### DISCUSSION TITLE
DoVi P5 conversion

### DISCUSSION THREAD

**Discussion by JellyRed** [quietvoid/dovi_tool#72] [Q&A] [answered]
Hello
I read documentation editor.md and have a simple question about converting p5 to p8.1 metadata.
I have p5 metadata from streaming service. I would like to add it to blu-ray remux. For example let's see latest DSNP IMAX releases.
Web-release has different picture size than remux. For example web-release has 69px letterbox bars, remux has 276px letterbox bars.
So how I should crop metadata to inject it to remux properly? Does p5 metadata have info about letterbox bars?
Usually I extract p5, add frames to match with remux, convert p5 to p8, add L5 info (add letterbox bars, set mode = 0, set "crop" flag as false).
I think I should add L5 info with 207px lettterbox bars and set "crop" flag as true.

**Comment by quietvoid** [comment, marked as answer]
The metadata in the IMAX version is not going to be accurate, because the metadata can be affected by parts of the picture that are not in the Blu-ray version.

Either way, you're supposed to keep the L5 metadata of the final video, which would be 276 in your example.

**Reply by JellyRed** [reply]
Thank you for your reply. I will set it to 276px.
Could you tell me am I right to use "crop" flag as true?
As I understand if I use "crop" flag as false I will set active area of movie. And after it meta will fit in into that area.
For example I have 3840x2160 video. It has 276px letterbox bars. So active area is 3840x1608.
If I set "crop" flag as true meta will fit in into 3840x1608. If I set "crop" flag as false meta will fit in into 3840x2160. Am I right?
Thanks in advance

**Reply by quietvoid** [reply]
I don't understand what you're saying.

It's simple: use crop only when the final video has no letterbox bars.
